creature_attributes.h

```diff
@@ -131,4 +133,5 @@ class CreatureAttributes {
   bool SERIAL(noAttackSound) = false;
   optional<CreatureId> SERIAL(creatureId);
   optional<double> SERIAL(moraleSpeedIncrease);
+  string SERIAL(deathDescription)="killed"_s;
```

Comment:
Make sure you add deathDescription to the serialization function.

Reply:
I will have to work out how that works.